docs/spec/reactors/block_sync/reactor.md

```diff
@@ -46,256 +46,256 @@ type bcStatusResponseMessage struct {
 
 ## Architecture and algorithm
 
-The Blockchain reactor is organised as a set of concurrent tasks: 
-    - Receive routine of Blockchain Reactor 
-    - Task for creating Requesters
-    - Set of Requesters tasks and 
-    - Controller task.        
+The Blockchain reactor is organised as a set of concurrent tasks: - Receive routine of Blockchain Reactor - Task for creating Requesters - Set of Requesters tasks and - Controller task.
 
 ![Blockchain Reactor Architecture Diagram](img/bc-reactor.png)
 
 ### Data structures
 
 These are the core data structures necessarily to provide the Blockchain Reactor logic.
 
-Requester data structure is used to track assignment of request for `block` at position `height` to a 
-peer with id equals to `peerID`. 
+Requester data structure is used to track assignment of request for `block` at position `height` to a
+peer with id equals to `peerID`.
 
 ```go
 type Requester {
-  mtx          Mutex		
+  mtx          Mutex
```

Comment:
oh man this spec needs to be re-written. Way too much implementation specific detail in here